docs/basics/userguide/seeding.md

```diff
@@ -85,7 +116,7 @@ env.reseed(seeder)  # seeder first use
 agent = UCBVIAgent(env)
 agent.reseed(seeder)  # same seeder
 
-# check that the generated numbers are differents
```

Comment:
I would still have used an s to different no ?

Reply:
my translator disagree... :/